### ISSUE TITLE
Conflict with native Firefox shortcut for taking screenshots

### ISSUE THREAD

**Issue by FreeYourMinds** [ericpromislow/tabhunter#139]
# Issue

Can not invoke the shortcut for Tabhunter on Windows

## Reason

`Ctrl-Shift-S` is Firefox native shortcut for taking a screenshot on Windows -> [Firefox Screenshots](https://support.mozilla.org/en-US/kb/how-do-i-create-screenshot-my-problem#w_firefox-screenshots)

Also, I cannot change TabHunters shortcut in the settings, but from my understanding this seem to be per design?

## Steps to reproduce

Try invoking TabHunters shortcut in Forefox on Windows

**Comment by ericpromislow** [comment]
No, it's just that no one on the team has found time to work on this

**Comment by FreeYourMinds** [comment]
I see, so there is currently no way of invoking TabHunter on Windows per hotkey.

Wouldn't it be an option to just change the default key binding for now?

**Comment by ericpromislow** [comment]
Right now no one on the "team" has a Windows development machine. Only macOS and linux. But the source code is right here and anyone is welcome to contribute.

**Comment by FreeYourMinds** [comment]
@ericpromislow as you wish PR #141 

***Note***: 

As I do not have a Ruby dev environment nor are there any instructions on how to set up a dev environment for windows, this is an untested change! I have only verified that the new shortcut is not taken by any others on windows per default. But as this is only a change in the manifest, I considered it an acceptable risk

**Comment by unleashed7** [comment]
You can change the shortcut in Extensions > Manage Extension Shortcuts to a different key.

**Comment by FreeYourMinds** [comment]
> You can change the shortcut in Extensions > Manage Extension Shortcuts to a different key.

Sadly you can not. That is part of the problem. Any Change is neither applied nor persisted.

**Comment by unleashed7** [comment]
Not sure what the issue is on your end, for me it works without a problem. Tried changing it to Ctrl+Shift+F and Ctrl+Shift+Space just to test and it opens with that hotkey.

https://github.com/user-attachments/assets/006b41a5-0ef5-48ae-b494-deaa2806e999

**Comment by FreeYourMinds** [comment]
> Not sure what the issue is on your end, for me it works without a problem. Tried changing it to Ctrl+Shift+F and Ctrl+Shift+Space just to test and it opens with that hotkey.
> 2025-01-03.01-05-49.mp4

Interesting, you were right, it is on my side. Using a fresh profile, this does work 👍 
Nevertheless, I would still change the default shortcut to something that is not in conflict with a native Firefox one

**Comment by clach04** [comment]
Interestingly the current default keymapping under Windows 11, `Ctrl-Shift-S` conflicts with the Operating System screen snapshot tool ().

With both Microsoft Edge and Google Chrome, the change in https://github.com/ericpromislow/tabhunter/pull/141 does not work for me.

It took me a rather long time to locate the keyboard shortcut options to override this! For (possibly only my) reference:

  * chrome://extensions/shortcuts
  * edge://extensions/shortcuts -- Edge can use either link

Manage Extension area does NOT cover this.

Presumably the PR not working is a v3 manifest issue? I've not had time to dig any further into this.

**Comment by clach04** [comment]
Explict Cross reference:

  * #119

**Comment by ericpromislow** [comment]
Nothing's changed - no one on the team has a windows machine.
